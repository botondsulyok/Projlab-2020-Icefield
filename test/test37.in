field spawn 3
entity esk Eskimo spawn
item shovel1 Shovel esk
stats spawn
stats esk
dig esk
stats spawn
stats esk
dig esk
stats spawn
stats esk
dig esk
stats spawn
stats esk