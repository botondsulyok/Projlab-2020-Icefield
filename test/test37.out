id: spawn
type: iceunit
building: plains0
snow: 3
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

id: esk
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: shovel1
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: spawn
type: iceunit
building: plains0
snow: 1
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

id: esk
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: shovel1
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

id: esk
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 2
digger: shovel1
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: esk
pullers: esk
masters: esk
neighbours: null

id: esk
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 2
digger: shovel1
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

